### ISSUE TITLE
problem with fancybox

### ISSUE THREAD

**Issue by HussamGasmalseed** [simogeo/Filemanager#543]
i am trying to use the file manager with fancybox, but i'm not able to select the image, just browsing ?

**Comment by simogeo** [comment]
I'll give you the advice to switch to https://github.com/servocoder/RichFilemanager

**Comment by psolom** [comment]
Right. You can find fancybox implementation example in the `demo` folder:

https://github.com/servocoder/RichFilemanager/tree/master/demo
